src/database/sql/audio.py

```diff
@@ -255,8 +278,9 @@ def get_bpm(self, audio_id: str) -> str:
 
         query = """
                 SELECT bpm, bpm_confidence, last_updated
-                FROM Audio 
-                WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+                FROM Audio
+                WHERE audio_release=:rel AND audio_side=:side
+                AND audio_track=:track
```

Comment:
If you want this AND `audio_track=:track` to be on a new line, please indent it.

Reply:
Removed